scripts/manageacl.py

```diff
@@ -2,11 +2,11 @@
 import asyncio
 import json
 import logging
-from typing import Union
+from typing import Any, Union
 
 from azure.core.credentials import AzureKeyCredential
 from azure.core.credentials_async import AsyncTokenCredential
-from azure.identity import AzureDeveloperCliCredential
+from azure.identity.aio import AzureDeveloperCliCredential
```

Comment:
This is one I want feedback on, as the code later says it is expecting an AsyncTokenCredential, but it wasn't pulling the async version of AZDCredential. So should it be sync or async?

Reply:
I think this was just a typo on the import

Reply:
Everything should be async

Reply:
Great example of type annotations catching bugs!